docs/quickstart/README.md

```diff
@@ -0,0 +1,298 @@
+Geotrellis Quickstart Guide
+===========================
+
+Geotrellis is a large and evolving package, which makes it difficult to maintain current documentation.  To make up for this deficiency, this quickstart guide will provide a set of documents describing case studies that accomplish some simple goals.  These examples are meant to highlight the fundamental Geotrellis classes as well as show some basic Scala techniques for those new to the language.
```

Comment:
Convention is to use line breaks after each sentence, so that diffs/comments can be made per sentence. I see you use double spaces for sentence separation, so this should quick change by just replacing `  ` with a new line.